src/Universum/Lifted/File.hs

```diff
@@ -49,4 +53,14 @@ openFile :: MonadIO m => FilePath -> IOMode -> m Handle
 openFile a b = liftIO (XIO.openFile a b)
 {-# INLINE openFile #-}
 
+-- | Close a file handle
+hClose :: MonadIO m => Handle -> m ()
+hClose = liftIO . hClose
```

Comment:
This looks like infinite recursion (`hClose` on the right is the same as on the left, AFAIU).

Reply:
Yes, you are correct. How embarrassing 😅